### PR TITLE
Adding state noteListBar

### PR DESCRIPTION
**Pull request by MayuBen1027**
Adding state noteListBar based on the information found [here](https://github.com/inkdropapp/docs/blob/master/src/pages/reference/state-note-list-bar.md).

### CODE REVIEW

File: src/app/states/note-list-bar/page.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+
+---
+
+## Data Structure
+
+<Row>
+  <Col>
+    <Properties>
+      <Property name="selectedNoteIds" type="string[]">
```
Comment (Keisir) [inline src/app/states/note-list-bar/page.mdx:19]:
```suggestion
      <Property name="selectedNoteIds" type="Array<string>">
```

File: src/app/states/note-list-bar/page.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        An array of note ID of selected notes on the note list bar.
+      </Property>
+      <Property name="actionTargetNoteIds" type="string[]">
```
Comment (Keisir) [inline src/app/states/note-list-bar/page.mdx:22]:
```suggestion
      <Property name="actionTargetNoteIds" type="Array<string>">
```

File: src/app/states/note-list-bar/page.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        An array of note ID of selected notes on the note list bar.
```
Comment (Keisir) [inline src/app/states/note-list-bar/page.mdx:23]:
```suggestion
        An array of note IDs of target notes for context menu on the note list bar.
```

**Review by Keisir — COMMENTED** [review]
I think the readability will be better by using "Array<T>" instead of using a shorthand operator.

**Review by Keisir — APPROVED** [review]
Thanks
LGTM

**Comment by craftzdog** [comment]
Thanks @MayuBen1027 and @Keisir for the effort!

**Pull request merged**